pkg/lib/robots/robots_txt.go

```diff
@@ -0,0 +1,23 @@
+package robots
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// Disallow all crawlers
+var robotsTxt = strings.Trim(`
+User-agent: *
+Disallow: /
+`, "\n")
```

Comment:
I suggest we remove `strings.Trim` and make the string content match the actual content we will return.

The body should be `User-agent: *\nDisallow: /\n` The trailing newline should be kept.